geometry/eolearn/tests/test_superpixel.py

```diff
@@ -52,10 +45,7 @@
                 max_num_iter=20,
                 sigma=0.8,
             ),
-            0,
-            48,
-            24.6072,
-            25,
+            {"exp_dtype": np.int64, "exp_min": 0, "exp_max": 48, "exp_mean": 24.6072, "exp_median": 25},
```

Comment:
not sure i'd include the expected dtype unless this is something that we really expect
if some other library changes something and now we by default output np.int32, should we be notified of that? We don't need values larger than int32

Reply:
nevermind, saw that it was already in the originals

Reply:
Resolved in chat